lib/ObjWriter/.nuget/Microsoft.DotNet.ObjectWriter.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Microsoft.DotNet.ObjectWriter</id>
-    <version>1.0.4-prerelease-00001</version>
+    <version>1.0.6-prerelease-00001</version>
```

Comment:
As commented below in the main reply, once it's merged, I will push it manually. But we don't need to update CoreRT which will automatically pick the latest one in case this postfix increment.